packages/clarity-js/src/data/metadata.ts

```diff
@@ -32,6 +32,11 @@ export function start(): void {
   dimension.log(Dimension.PageTitle, title);
   dimension.log(Dimension.Url, location.href);
   dimension.log(Dimension.Referrer, document.referrer);
+  dimension.log(Dimension.Language, (<any>navigator).userLanguage || navigator.language);
+  dimension.log(Dimension.ScreenWidth, `${screen.width || Constant.Empty}`);
```

Comment:
is this different from the viewport width and height that we already capture?

Reply:
yes actually, they are constant for a monitor rather than specific to a window's size